stern/stern.go

```diff
@@ -48,6 +49,36 @@ func Run(ctx context.Context, client kubernetes.Interface, config *Config) error
 		}
 	}
 
+	newTailOptions := func() *TailOptions {
+		return &TailOptions{
+			Timestamps:      config.Timestamps,
+			TimestampFormat: config.TimestampFormat,
+			Location:        config.Location,
+			SinceSeconds:    ptr.To[int64](int64(config.Since.Seconds())),
+			Exclude:         config.Exclude,
+			Include:         config.Include,
+			Highlight:       config.Highlight,
+			Namespace:       config.AllNamespaces || len(namespaces) > 1,
+			TailLines:       config.TailLines,
+			Follow:          config.Follow,
+			OnlyLogLines:    config.OnlyLogLines,
+		}
+	}
+	newTail := func(t *Target) *Tail {
+		return NewTail(client.CoreV1(), t.Node, t.Namespace, t.Pod, t.Container, config.Template, config.Out, config.ErrOut, newTailOptions())
+	}
+
+	if config.Stdin {
+		tail := NewFileTail(config.Template, "stdin", os.Stdin, config.Out, config.ErrOut, newTailOptions())
+		var eg errgroup.Group
+		eg.SetLimit(config.MaxLogRequests)
+		eg.Go(func() error {
+			defer tail.Close()
+			return tail.Start()
+		})
+		return eg.Wait()
```

Comment:
It seems --stdin is always single input, so I think we do not need to use errgroup.

```suggestion
		return tail.Start()
```